### DISCUSSION TITLE
Docker image for 5.x

### DISCUSSION THREAD

**Discussion by georgkrause** [WeblateOrg/weblate#9818] [Q&A] [answered]
Hello,

at first, thank you to the whole Team and community for the great, reliable tool! <3

I just noticed there is a new major release in version 5, but the images are not available on docker hub. 

Do you plan to provide those images or did you stopped providing them with 5.x?
I also have the intention those images are pushed manually, do you have plans to automate this? Is there a way to help you doing that?

Kind regards, Georg

**Comment by nijel** [comment, marked as answer]
It mostly automated, just the tag needs to be created manually, what I've done just now. It took a bit longer than usual because I'm travelling just after the release.

**Reply by georgkrause** [reply]
@nijel Late but still: Thank you a lot! :)
